documentation/STEPS.md

```diff
@@ -236,6 +238,35 @@ Key       | Type   | Required | Supports Variable | Description
 `message` | String | Yes      | Yes               | The optional message to fail with.
 
 
+---
+
+### include_test
+
+**How it Works**
+
+1. Import and execute all the steps from another test given its name and its suite
+
+**Example**
+
+```json
+{
+  "id": "include_test",
+  "image": "<optional_base_64_image>",
+  "values": {
+    "suiteName": "<optional_string>",
+    "testName": "myTestName",
+  }
+}
+```
+
+**Values**
+
+Key         | Type   | Required | Supports Variable | Description
+----------  |--------|----------|-------------------|-------------
+`suiteName` | String | No       | No                | The optional suite to which the `testName` belongs.
+`testName`  | String | Yes      | No                | The name of the test to import.
```

Comment:
Should have a required `version` too.

Reply:
Or at least a non-required `version` that specifies that the highest version will be picked.  That's ok too.